go/kbfs/libkbfs/prefetcher.go

```diff
@@ -644,6 +644,33 @@ func (p *blockPrefetcher) request(ctx context.Context, priority int,
 	return 0, 0, 0
 }
 
+func (p *blockPrefetcher) handleStatusRequest(req *prefetchStatusRequest) {
+	pre, isPrefetchWaiting := p.prefetches[req.ptr.ID]
+	if !isPrefetchWaiting {
+		req.ch <- PrefetchProgress{}
+	} else {
+		req.ch <- pre.PrefetchProgress
+	}
+}
+
+// handleCriticalRequests should be called periodically during any
+// long prefetch requests, to make sure we handle critical requests
+// quickly.  These are requests that are required to be run in the
+// main processing goroutine, but shouldn'won't interfere with
```

Comment:
shouldn'won't is a new one. Is that 85% or 95% certainty?

Reply:
Oops will fix, thanks.